test/models/friend_request_test.rb

```diff
@@ -0,0 +1,12 @@
+require 'test_helper'
+
+class FriendRequestTest < ActiveSupport::TestCase
```

Comment:
This test is incomplete as it is. Please test for:

1. Valid friend_requests (see rails tutorial examples for how to write tests properly and completely).

2. Invalid friend_requests.

These two basically are also the association tests. Basically a valid request would have a receiver_id and a sender_id. The converse would be the invalid request